### ISSUE TITLE
Someone is using your project to sell it as a token

### ISSUE THREAD

**Issue by yzthink** [mlfoundations/open_lm#247]
Hello gentlemen, I would like to ask, do you know that someone is using your project to sell it as a token to raise public funds? I would like to know if you are aware of this matter and what do you think? And there are plans to deploy your model on the blockchain for charging. Is this what you expect? This is their Twitter and I want to know what you think.@OraProtocol
![image](https://github.com/mlfoundations/open_lm/assets/166803373/fbf6ea8f-04ca-4548-936f-32873a109bfd)

**Comment by achalddave** [comment]
No, we were not aware of this. We are not affiliated with this organization and have no association with the token.
